src/main/java/de/tum/in/test/api/ast/type/ConditionalType.java

```diff
@@ -0,0 +1,37 @@
+package de.tum.in.test.api.ast.type;
+
+import java.util.Map;
+
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.ast.stmt.IfStmt;
+import com.github.javaparser.ast.stmt.SwitchStmt;
+
+public enum ConditionalType implements Type {
```

Comment:
Any ideas regarding the expression version, ternary operator and switch expression? The statements can easily be replaced by the expression versions, not sure what we want here.

Reply:
[Implemented part of the third feedback iteration](https://github.com/ls1intum/Ares/pull/248/commits/625a5f2b6d342cb4cbe499e7ea8e9d197d7c5c96): Tenary operators and switch expressions are now covered.